pkg/controller/ingress/create.go

```diff
@@ -361,6 +373,10 @@ func (lbc *EngressController) createNodePortPods() error {
 			Name:      VoyagerPrefix + lbc.Config.Name,
 			Namespace: lbc.Config.Namespace,
 			Labels:    labelsFor(lbc.Config.Name),
+			Annotations: map[string]string{
+				LoadBalancerSourceType: lbc.kind,
```

Comment:
existing ingress resources will be updated if accordingly if the annotations are not found. this was fixed in commit `9d7631d`